google/auth/compute_engine/_metadata.py

```diff
@@ -294,7 +294,7 @@ def get_universe_domain(request):
             404 occurs while retrieving metadata.
     """
     universe_domain = get(
-        request, "universe/universe_domain", return_none_for_not_found_error=True
```

Comment:
Do we have to support both endpoints right now or we can just try only the new one?